### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by asci**
## What
this is initial commit for polyfill repo

## What
- please check the readme
- source code
- other files required for Travix OSS

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "ecmaVersion": 5
+  },
+  "extends": [
+    "travix/base"
+  ],
+  "env": {
+    "browser": true
+  },
+  "globals": {
+    "__base": true,
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: Doesn't seem you're using this anywhere.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "assert": true,
+    "beforeEach": true,
+    "describe": true,
+    "enzyme": true,
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: `enzyme` is not a dependency, so please remove it.

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "extends": [
+    "travix/base"
+  ],
+  "env": {
+    "browser": true
+  },
+  "globals": {
+    "__base": true,
+    "assert": true,
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: `assert`is not a dependency nor provided by any dependency, so please remove it.

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "beforeEach": true,
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: `beforeEach`is not a dependency nor provided by any dependency, so please remove it.

File: .eslintrc [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    "describe": true,
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: `describe`is not a dependency nor provided by any dependency, so please remove it.

File: .eslintrc [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    "enzyme": true,
+    "expect": true,
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: `expect`is not a dependency nor provided by any dependency, so please remove it.

File: .eslintrc [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    "it": true,
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: `it`is not a dependency nor provided by any dependency, so please remove it.

File: .eslintrc [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    "jest": true
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: `jest`is not a dependency so please remove it.

File: .eslintrc [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "rules": {
+    "no-var": ["off"],
+    "vars-on-top": ["off"],
+    "no-extra-semi": ["off"],
+    "no-console": ["off"],
+    "comma-dangle": ["error", "always-multiline"],
+    "max-len": [2, {"code": 120, "comments": 120, "tabWidth": 4}],
```
Comment (mAiNiNfEcTiOn) [inline .eslintrc]: isn't the `tabWidth` conflicting with the `ident_size` of the `.editorconfig`?

File: package.json [hunk 10]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "css-themes-polyfill",
+  "version": "0.1.0",
+  "description": "Travix CSS variables for themes polyfill",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "node test/index.spec.js",
```
Comment (mAiNiNfEcTiOn) [inline package.json]: Have you considered adding a `prepare` task to minify/uglify the javascript code and provide the polyfill as minified IIFE?

File: package.json [hunk 11]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "css-themes-polyfill",
```
Comment (fahad19) [inline package.json]: isn't it a CSS Variables Polyfill?

Comment (fahad19) [inline package.json]: there's no rule of any kind, but unless the project happens to have its own name, we usually prefix the package names with `travix-`.

File: package.json (hunk 10 above)
Comment (asci) [inline package.json]: I haven't but it sounds reasonable, will add. Thanks!

**Comment by asci** [comment]
Updated, thanks for comments @fahad19 @mAiNiNfEcTiOn

**Pull request merged**